lms/static/sass/views/_program-details.scss

```diff
@@ -393,14 +397,7 @@
     }
 
     .run-select {
-      @include margin-right(10px);
-
-      width: 95%;
-
-      @media (min-width: $bp-screen-sm) {
-        width: 300px;
-      }
-
```

Comment:
These fixes are for a bug with the styling for the run selection that exists today on Prod. See before and after images below.

Reply:
Nice Shoe in here! 👍